wix-bookings/guides/Terminology.md

```diff
@@ -1,58 +1,57 @@
+| Slot | A time period available for booking by a customer for a service. In appointment services, this encompasses all available booking times. For classes, it refers to all existing sessions open for booking. Refer to appointment schedule for details on appointment services and class schedule for details on classes. |
+| [Service Provider Interface (SPI)](https://dev.wix.com/api/rest/getting-started/service-provider-interface#getting-started_service-provider-interface_introduction) | SPIs are APIs that are defined by Wix, which you can choose to implement. By doing so, you become a service provider. Then, Wix calls your service during a certain flow, waits for your response, and continues the flow with it. For example, as a pricing provider you can offer custom varied pricing options to a Wix site using the [Wix Bookings Pricing Integration REST SPI](https://dev.wix.com/api/rest/wix-bookings/pricing-integration-spi). |
+| Site owner | The owner of the Wix site which has Wix Bookings installed and all [site contributors](https://support.wix.com/en/article/inviting-people-to-contribute-to-your-site). |
+| Suggested participant (Waitlist) | When a session is fully booked, customers can join the waitlist. If a spot opens up due to a cancellation, the first person on the waitlist (the suggested participant) is notified and given a chance to book the session within a specified time. If they don't book it, the opportunity is passed to the next person on the list. |
+| Variant (Pricing) | Unique combination of choices for a service. For example: `{"time": "afternoon", "ageGroup": "adult", "equipment": "kettlebells"}`. Each variant may have a different price. |
+| [Varied pricing](https://support.wix.com/en/article/wix-bookings-understanding-price-options) | The ability to charge for services based on different factors, such as the staff member who provides the service or the equipment needed.  |
+| Visitor | Anyone who isn't registered as a member or hasn't logged in to the Wix site. If a visitor creates a booking they're added as a [contact](https://dev.wix.com/docs/rest/crm/members-contacts/contacts/introduction). |
+| `WAITING_LIST` (Booking status) | Indicates that the booking is on a waitlist. Currently, waitlists are available only for classes and not for courses or appointment services. |
+| Waitlist | A feature available for class services, allowing customers to request to be added to a fully booked session in case of cancellations. Not available for courses or appointment services. |
+| Waitlist time window | The duration that's given to the suggested participant to book a place in the session before it's offered to the next customer on the waitlist. |
+| Membership plans with [Wix Pricing Plans](https://support.wix.com/en/article/wix-bookings-about-wix-bookings#selling-membership-plans-and-packages) | Prepaid bundles of services or memberships providing access to certain services. |
```

Comment:
not sorted

Reply:
I've shortened it "Wix Pricing Plans", so it's in alphabetical order.